frontend/src/components/Header/Header.tsx

```
@@ -1,16 +1,56 @@
+          </button>
+        </div>
       </div>
-      <Button variant="default" className="order-1">
-        Press Me
-      </Button>
-    </div>
+
+      {/* Mobile Menu Dropdown (Includes Login) */}
+      {isOpen && (
+        <div className="md:hidden dark:bg-gray-800 p-4 space-y-2">
+          <a href="#" className="block py-2">Home</a>
```

Biome (1.9.4)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)